kie-dmn-core/src/test/resources/org/kie/dmn/core/priority_table_context_recursion.dmn

```diff
@@ -0,0 +1,102 @@
+  <semantic:extensionElements/>
+  <semantic:itemDefinition label="tRiskCategory" name="tRiskCategory">
+    <semantic:typeRef>feel:string</semantic:typeRef>
+    <semantic:allowedValues triso:constraintsType="enumeration">
+      <semantic:text>"High","Low","Medium"</semantic:text>
+    </semantic:allowedValues>
+  </semantic:itemDefinition>
+  <semantic:itemDefinition label="tApprovalStatus" name="tApprovalStatus">
+    <semantic:typeRef>feel:string</semantic:typeRef>
+    <semantic:allowedValues triso:constraintsType="enumeration">
+      <semantic:text>"Approved","Declined","Referred"</semantic:text>
+    </semantic:allowedValues>
+  </semantic:itemDefinition>
+  <semantic:decision id="_3b2953a3-745f-4d2e-b55d-75c8c5ae653c" name="Approval Status" triso:displayName="Approval Status">
+    <semantic:variable id="_7fe30427-4140-4e53-a070-2ecfc723d82b" name="Approval Status" typeRef="tApprovalStatus"/>
+    <semantic:informationRequirement>
+      <semantic:requiredInput href="#_5a4bdb64-f0ef-4978-9e03-6f1ae64a1f17"/>
+    </semantic:informationRequirement>
+    <semantic:informationRequirement>
+      <semantic:requiredInput href="#_8ff18665-84e9-49f2-a8df-8981b1844549"/>
+    </semantic:informationRequirement>
+    <semantic:context>
+        <semantic:contextEntry>
+            <semantic:variable id="m1" name="m1" typeRef="feel:string"/>
+            <semantic:literalExpression>
+                <text>"Hello World"</text>
+            </semantic:literalExpression>
+        </semantic:contextEntry>
+        <semantic:contextEntry>
+            <semantic:context>
+                <semantic:contextEntry>
+                    <semantic:decisionTable hitPolicy="PRIORITY" id="_c98573c0-08b5-4bc3-baf1-3b065fedf1d1" outputLabel="Approval Status">
+                      <semantic:input id="_0524c2a9-a223-425a-988f-72d39eed9714" label="isAffordable">
+                        <semantic:inputExpression typeRef="feel:boolean">
+                          <semantic:text>isAffordable</semantic:text>
+                        </semantic:inputExpression>
+                      </semantic:input>
+                      <semantic:input id="_705b772e-6ac9-4c09-ace8-5165438454d9" label="RiskCategory">
+                        <semantic:inputExpression typeRef="tRiskCategory">
+                          <semantic:text>RiskCategory</semantic:text>
+                        </semantic:inputExpression>
+                      </semantic:input>
+                      <semantic:output id="_704c6117-4670-4d2f-865e-a4d973db5e9f" typeref="ts:tApprovalStatus" />
```

Comment:
Removed.
This line 45 I've noticed is actually the "spurious error" notice `typeref` has lower case `r`, so it was really testing the recursion anyway